Project Management Process.md

```diff
@@ -0,0 +1,287 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
```

Comment:
Saying that something is `opt-in` and `not required` should not be the same as `you don't have to buy in into the process, but if you want to understand how to contribute, you have to buy in`